tasks/verify-access-to-resources/verify-access-to-resources.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: verify-access-to-resources  
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to verify access to required resources & check if internal requests can be created.
+  params:
+    - name: release
+      description:
+        The namespaced name (namespace/name) of the Release custom resource
```

Comment:
These descriptions should match what you have in the README (I would vote to copy the ones from there to here as the capitalization of the resources is not consistent in the descriptions in this file)